cockatrice/src/deck_loader.cpp

```diff
@@ -103,6 +103,47 @@ bool DeckLoader::saveToFile(const QString &fileName, FileFormat fmt)
     return result;
 }
 
+struct FormatDeckListForExport
+{
+    QString &mainBoardCards;
+    QString &sideBoardCards;
+    
+    FormatDeckListForExport(
+                    QString &_mainBoardCards, QString &_sideBoardCards
+                            ) : mainBoardCards(_mainBoardCards),
+                                sideBoardCards(_sideBoardCards){};
+    
+    void operator()(const InnerDecklistNode *node, const DecklistCardNode *card) const{
+        CardInfo * dbCard = db->getCard(card->getName());
+        if (!dbCard || dbCard->getIsToken())
+            return;
+        if(node->getName() == DECK_ZONE_SIDE){
+            sideBoardCards+=QString::number(card->getNumber());
+            sideBoardCards+="%20";
+            sideBoardCards+=card->getName();
```

Comment:
I think this needs to be url escaped

Reply:
Agreed

Reply:
URL is escaped in the view area, could move it to this function if desired.